api/nodes/serializers.py

```diff
@@ -1142,11 +1142,19 @@ def create(self, validated_data):
         return self.make_instance_obj(node)
 
 
+class RegistrationSchemaRelationshipField(RelationshipField):
+
+    def to_internal_value(self, registration_schema_id):
+        schema = get_object_or_error(RegistrationSchema, registration_schema_id, self.context['request'])
+        if schema.schema_version != LATEST_SCHEMA_VERSION or not schema.active:
+            raise exceptions.ValidationError('Registration supplement must be an active schema.')
+        return {'registration_schema': schema}
+
+
 class DraftRegistrationSerializer(JSONAPISerializer):
 
     id = IDField(source='_id', read_only=True)
     type = TypeField()
-    registration_supplement = ser.CharField(source='registration_schema._id', required=True)
```

Comment:
`registration_supplement` removed here instead of creating a new version, because we aren't using this endpoint yet, and I suspect few others are. However, if we have a hard rule on this, I'll create a version.

Reply:
This should be fine for Embosf. We only use this property to create (it's never read). We'll clean up after this PR lands.